src/components/modal/styles/CdrModal.module.scss

```diff
@@ -17,7 +17,7 @@ $modal-animation-duration: 150ms;
   right: 0;
   top: 0;
   visibility: visible;
-  z-index: 1000;
+  z-index: 9999;
```

Comment:
Getting this above gnav so when teleport happens it doesn't end up underneath it.